### PR TITLE
Improves the existing documentation

### LINKED ISSUE #2: Create a basic documentation for the project with sphinx.
* [x] Add Alba as Author
* [x] Add Installation section
* [x] Add Contributing section
* [x] Add Authors
* [ ] Add usage section
* [ ] Add examples
* [ ] Document the git workflow in the wiki
* [ ] Document the release process in the wiki
* [ ] Documentation for sponsors
* [ ] Documentation for researchers

### PR DESCRIPTION
**Pull request by jloehel**
- Adds Alba as an author

Fixes: #2 

Signed-off-by: Jürgen Löhel <juergen@loehel.de>

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -0,0 +1,85 @@
+============
+Contributing
+============
+
+Contributions are welcome, and they are greatly appreciated! Every
```
Comment (alba-vcl) [inline CONTRIBUTING.rst:5]: "Each one helps"   or something different instead of "Every little bit helps"

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+    git clone git@github.com:jloehel/molden_modifier.git
+
+3. Create a branch for local development::
+
+    git checkout -b name-of-your-bugfix-or-feature
+
+   Now you can make your changes locally.
+
+4. When you're done making changes run all the checks and docs builder with `tox <https://tox.readthedocs.io/en/latest/install.html>`_ one command::
```
Comment (alba-vcl) [inline CONTRIBUTING.rst:51]: you are

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+    tox
+
+5. Commit your changes and push your branch to GitHub::
+
+    git add .
```
Comment (alba-vcl) [inline CONTRIBUTING.rst:57]: should the dot be there?

File: CONTRIBUTING.rst (hunk 1 above)
Comment (jloehel) [reply CONTRIBUTING.rst:5]: Okay, I'll change it later.

File: CONTRIBUTING.rst (hunk 2 above)
Comment (jloehel) [reply CONTRIBUTING.rst:51]: You are right, I should not shorten it.

File: CONTRIBUTING.rst (hunk 3 above)
Comment (jloehel) [reply CONTRIBUTING.rst:57]: Yes, the dot is necessary.

**Pull request closed without merging**